webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -394,6 +394,9 @@ public function viewAction(
                     '/\[output storage truncated after \d* B\]/',
                     (string)$runResult['output_run_last_bytes']
                 );
+                if ($firstJudgingRun) {
+                   $runResult['testcasedir'] = $firstJudgingRun->getTestcaseDir();
```

Comment:
```suggestion
                    $runResult['testcasedir'] = $firstJudgingRun->getTestcaseDir();
```

nitpick: phpcs wants 4 spaces here.